src/static/images/collection-items/aircraft-carrier.dzi

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
the dzi file itself is actually an XML file, not an image file. Had to do bit of reading bout it too to realize that's how it's supposed to work. You can open the file in Atom (it won't crash) and actually see the XML attributes. After the client calls the `.dzi` file, it immediately looks into the same location (i.e. `/static/images/collection-items/..`) for the tiles. and renders them accordingly.